grpc/src/client/subchannel.rs

```diff
@@ -388,7 +400,9 @@ impl InternalSubchannel {
             // error string containing information about why the connection
             // terminated? But what can we do with that error other than logging
             // it, which the transport can do as well?
-            svc.disconnected().await;
+            if let Err(e) = closed_rx.await {
```

Comment:
Should the above task spawn be on the runtime as well instead of directly using `tokio`?